src/main/kotlin/com/ktinsta/server/model/Post.kt

```diff
@@ -0,0 +1,35 @@
+package com.ktinsta.server.model
+
+import org.springframework.format.annotation.DateTimeFormat
+import java.time.Instant
+import java.util.*
+import javax.persistence.*
+import javax.validation.constraints.NotBlank
+
+@Entity
+data class Post(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var id: Long = 0,
+
+    //TODO: Прописать каскадное удаление
+    @ManyToOne(targetEntity = User::class)
```

Comment:
Для каскадных операций - 
```suggestion
    @ManyToOne(targetEntity = User::class, cascade=CascadeType.REMOVE)
```